sphinx/util/nodes.py

```diff
@@ -96,17 +104,30 @@ def is_translatable(node):
     if isinstance(node, addnodes.translatable):
         return True
 
+    if isinstance(node, nodes.Inline) and 'translatable' not in node:
+        # inline node must not be translated if 'translatable' is not set
+        return False
```

Comment:
This might not be related with this PR. Is this right?

Reply:
It relates this PR, sorry for my less comment on this change.

Previously, on line 116, the code returned False under the same conditions. Now ton of log outputs are emitted on line 117 for each Inline nodes, so I divided this condition into separated "if" statement.

Reply:
Okay, I understand.